setup_prow_ci_env.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/bash
```

Comment:
This is not a shell script

Reply:
Doh. Need more coffee.

Reply:
My personal preference is for this stuff to go in the Makefile directly. I'll defer to @YuviGold but having to find the make recipe then the shell script being called gives me a headache.